app/models/movie.rb

```diff
@@ -1,3 +1,33 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  validates :title, presence: true
+  validates :creation_year, presence: true, numericality: true
+  validates :genre, presence: true
+
+  def concat_movie_details
+    "#{title} - #{creation_year} - #{genre}"
+  end
+
+  def order_actors
+    actors.order(:age)
+  end
+
+  def average_age
+    ages = []
+    actors.each do |actor|
+      ages << actor.age
+    end
+    average = ages.sum / ages.length.to_f.round(1)
+    average
+  end
+
+  def add_actor(actor)
+    if !actors.include?(actor)
```

Comment:
Does the user story specify that only one instance of an actor is allowed to be in this movie? What about "The Wizard of Oz" or "Austin Powers"? Those actors may be one person but they had multiple roles... 🤔 food for thought. 

In any case, this line is not needed. It's not currently up to us (given the user stories) if we need this check, so don't include it until we have to.